### DISCUSSION TITLE
my friend account had been flagged spam

### DISCUSSION THREAD

**Discussion by Abhi05073** [community/community#111786] [New to GitHub] [answered]
my friend account had been flagged spam he is not able to do anything with github, he had raised the ticket 15 days ago but github did not respond,please  help us to resolve this issue, @githubsupport 
username of my friend account is @hacanand

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Abhi05073, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
